revisions/F0/src/constants/charmaps/credits_codepoints.asm

```diff
@@ -1,7 +1,10 @@
+;
+; Mapping from Unicode to the codepoints used in the ROM
+;
```

Comment:
I think I'd like to keep the `charmaps/credits_charmap.asm` charmap suffix for the filename.

I understand codepoint is the more precise technical term. But on the other hand, "charmap" is a widely used term, and the actual name of the assembler command. Plus it matches the constant name.

Reply:
I renamed this file so that there would be consistent names for these files which do similar things:
```
src/constants/charmaps/
- dialog_codepoints.asm
- credits_codepoints.asm
- name_codepoints.asm
```

And I gave those names so it would be clear how they relate to these files:
```
src/data/
- codepoint_to_diacritic.asm
- codepoint_to_tile.asm
- credits_staff_codepoint_to_diacritic.asm
```

I'm open to keeping the old name, but I'm not sure how best to re-label everything in that case.

Reply:
Hmm, I think `codepoint_to_*.asm` still makes sense. I kept those, and added extra comments to cross-reference the symbols, so that they're easier to find. What do you think?